clicommand/pipeline_upload.go

```diff
@@ -398,3 +462,22 @@ func searchForSecrets(l logger.Logger, cfg *PipelineUploadConfig, environ map[st
 		}
 	}
 }
+
+func validateJWK(key jwk.Key, alg jwa.SignatureAlgorithm) error {
+	validKeyTypes := []jwa.KeyType{jwa.RSA, jwa.EC, jwa.OctetSeq}
```

Comment:
Add a TODO to investigate OKP?

Reply:
so it turns out that OKP is Ed25519! easy peasy, i'll add that in today.